src/aliases.rs

```diff
@@ -49,5 +49,17 @@ pub type U2048 = Uint<2048, 32>;
 /// [`Uint`] for `4096` bits.
 pub type U4096 = Uint<4096, 64>;
 
+/// [`Bits`] for `128` bits.
+pub type H128 = Bits<128, 2>;
+
+/// [`Bits`] for `160` bits.
+pub type H160 = Bits<160, 3>;
+
+/// [`Bits`] for `256` bits.
+pub type H256 = Bits<256, 4>;
+
+/// [`Bits`] for `512` bits.
+pub type H512 = Bits<512, 8>;
+
 // TODO: B0, B1, B8, ... B4096
```

Comment:
We  should implement all of these. probably using a macro

Reply:
oops, turns out that's what `B` is about 🫡